httpmw/Ip_cognito_auth.go

```diff
@@ -1,18 +1,20 @@
 package httpmw
 
 import (
+	"log"
+	"strings"
+
 	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
 	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider/cognitoidentityprovideriface"
 	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt"
 	"github.com/protsack-stephan/gin-toolkit/httperr"
-	"log"
-	"strings"
 )
 
 // IpCognitoAuth middleware for:
 // * IP ranges verification in format "192.168.10.1-192.168.10.10,192.168.90.1-192.168.90.10"
 // * cognito authentication through Authorization Bearer Token
-func IpCognitoAuth(ipRange string, svc cognitoidentityprovideriface.CognitoIdentityProviderAPI) gin.HandlerFunc {
+func IpCognitoAuth(ipRange string, svc cognitoidentityprovideriface.CognitoIdentityProviderAPI, cogntoClientID string) gin.HandlerFunc {
```

Comment:
We can change this to just `clientID` we can infer that this is for `cognito` by the name of the function.